### PR TITLE
add Location field

### PR DESCRIPTION
**Pull request by willemarcel**
screenshot-2018-1-16 osm changeset analyzer

## What's missing:
- [x] When loading a filter link, its geometry is not being show on the map
- [x] Put the BBox field back
- [x] Add button to clear the Location

Solves https://github.com/mapbox/osmcha-frontend/issues/208 and https://github.com/mapbox/osmcha-frontend/issues/133

### CODE REVIEW

**Comment by willemarcel** [comment]
I removed the BBox field initially, but I think it's better to put it back as some users may need it (example https://github.com/mapbox/osmcha-frontend/issues/252)
I have one idea to integrate both fields in just one map.

**Comment by willemarcel** [comment]
All issues have been solved! We can start to test it, @tridip1931

I would like to have both the bbox and the location fields managed by only one map, but I didn't succeed to do that. Any idea on how to do that, @kepta ?

File: src/components/filters/location.js [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+        map.addControl(this.draw);
+
+        map.on('draw.create', this.drawingUpdate);
+        map.on('draw.modechange', this.clearBeforeDraw);
+        map.on('draw.delete', this.drawingUpdate);
+        map.on('draw.update', this.drawingUpdate);
+        map.on('style.load', () => {
+          if (this.props.value.get('0').get('value')) {
+            this.updateMap(this.props.value.get('0').get('value').toJS());
+          }
```
Comment (tridip1931) [inline src/components/filters/location.js]: 👋  @willemarcel, I am getting `TypeError: Cannot read property 'get' of undefined` 
on this.

**Comment by willemarcel** [comment]
@tridip1931 I fixed the error you pointed out. Thanks!!

**Comment by willemarcel** [comment]
I just discovered a bug when saving an AOI with the location geometry... working on this now

**Comment by willemarcel** [comment]
> I just discovered a bug when saving an AOI with the location geometry... working on this now

Solved! Now I think everything is working properly. @tridip1931 could you do some tests and review?

**Comment by willemarcel** [comment]
thanks @tridip1931 !

**Pull request merged**